src/components/hooks/use-firebase-fetch-data.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from 'react'
+import { initializeApp } from 'firebase/app'
+import { getDatabase, ref, onValue } from 'firebase/database'
+import { firebaseConfig } from 'common/constants'
+import useRegion from 'components/hooks/use-region'
+
+const useFetchFirebaseData = () => {
+    const [data, setData] = useState(null)
+    const [error, setError] = useState(null)
+    const { is_eu } = useRegion()
+
+    useEffect(() => {
+        const app = initializeApp(firebaseConfig)
+        const db = getDatabase(app)
+
+        const commoditiesRef = is_eu ? ref(db, 'eu/market') : ref(db, 'row/market')
```

Comment:
Do you think we should switch the turnery to inside the ref?

```suggestion
        const commoditiesRef = ref(db, is_eu ? 'eu/market' : 'row/market')
```